Add installation guide (#100)
pwntester/ysoserial.net: README.md

This software is a personal project and not related with any companies, including Project owner and contributors employers.

## Installation

> You can install the software from this page: https://github.com/pwntester/ysoserial.net/releases

Reviewer: ```suggestion
You can install the latest version and previous releases of YSoSerial.NET from [the releases page](https://github.com/pwntester/ysoserial.net/releases)
```

> - Release 1.34: [Download](https://github.com/pwntester/ysoserial.net/releases/download/v1.34/ysoserial-1.34.zip) (latest version)
> - Release 1.33: [Download](https://github.com/pwntester/ysoserial.net/releases/download/v1.33/ysoserial-1.33.zip)
> - Release 1.32: [Download](https://github.com/pwntester/ysoserial.net/releases/download/v1.32/ysoserial-1.32.zip)
> - Release 1.31: [Download](https://github.com/pwntester/ysoserial.net/archive/v1.31.zip)
> 

Reviewer: ```suggestion
```

> - Release 1.34: [Download](https://github.com/pwntester/ysoserial.net/releases/download/v1.34/ysoserial-1.34.zip) (latest version)
> - Release 1.33: [Download](https://github.com/pwntester/ysoserial.net/releases/download/v1.33/ysoserial-1.33.zip)
> - Release 1.32: [Download](https://github.com/pwntester/ysoserial.net/releases/download/v1.32/ysoserial-1.32.zip)
> - Release 1.31: [Download](https://github.com/pwntester/ysoserial.net/archive/v1.31.zip)
> 

Reviewer: ```suggestion
- Release 1.34 (latest version): [Download](https://github.com/pwntester/ysoserial.net/releases/download/v1.34/ysoserial-1.34.zip)

```
